commit 6ec0d58fbff7c1a814a21cf8c7fe0ef5106a04c1
Author: Steve Dickson <steved@redhat.com>
Date:   Wed Jan 20 14:16:08 2016 -0500

    mountd: print an error message when no versions are specified
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/mountd/mountd.c b/utils/mountd/mountd.c
index 8aca181..b584afc 100644
--- a/utils/mountd/mountd.c
+++ b/utils/mountd/mountd.c
@@ -794,9 +794,10 @@ main(int argc, char **argv)
 		}
 
 	/* No more arguments allowed. */
-	if (optind != argc || !version_any())
+	if (optind != argc || !version_any()) {
+		fprintf(stderr, "%s: No protocol versions specified!\n", progname); 
 		usage(progname, 1);
-
+	}
 	if (chdir(state_dir)) {
 		fprintf(stderr, "%s: chdir(%s) failed: %s\n",
 			progname, state_dir, strerror(errno));
